tests/README_INSTALL_TESTS.md

```
@@ -0,0 +1,24 @@
+# Running Installation Tests
+The tests in 	est_install_in_fresh_environment.py verify that the package dependencies are correctly specified.
+## Important: These tests MUST run serially
+These tests create virtual environments and install packages, which can cause issues when run in parallel.
+## How to run these tests:
+`ash
+# Run all installation tests (serially)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)